express/properties/workflow.py

```diff
@@ -1,6 +1,178 @@
 from express.properties import BaseProperty
+import os
+import copy
+from typing import List
 
 
+# Note for naming convention: code uses snake_case, JSON uses camelCase
+
+
+# Todo: This is the quick implementation of ExabyteML, and will be depreciated eventually
+class MLQuickImplementation(BaseProperty):
+    """
+    Quick implementation of the new version of ExabyteML
+    """
+
+    def __init__(self, name, parser, *args, **kwargs):
+        """
+        Constructor for MLQuickImplementation
+
+        Args:
+            name (str): Name of the workflow
+            parser (str): Parser to use with this workflow
+        """
+        super().__init__(name, parser, *args, **kwargs)
+        self.name = name
+        self.work_dir = self.kwargs["work_dir"]
+
+        object_storage_data = self.kwargs["object_storage_data"]
+        self.obj_storage_container = object_storage_data["CONTAINER"]
+        self.obj_storage_container_region = object_storage_data["REGION"]
+        self.obj_storage_container_provider = object_storage_data["PROVIDER"]
+        self.ml_cache_dir = ".mlcache"
```

Comment:
This is bad practice - you are relying on the name set inside a python script in an editable workflow unit file

Reply:
Per conversation this afternoon, this variable now gets set in Rupy and passed in. See PR in https://github.com/Exabyte-io/exabyte-rupy/pull/318